packages/theme/src/cli/utilities/theme-environment/storefront-renderer.ts

```diff
@@ -109,15 +104,15 @@ function buildStorefrontUrl(session: DevServerSession, {path, sectionId, query}:
   return `${url}?${params}`
 }
 
-function buildBaseStorefrontUrl(session: DevServerSession) {
+export function buildBaseStorefrontUrl(session: AdminSession) {
   if (isThemeAccessSession(session)) {
     return 'https://theme-kit-access.shopifyapps.com/cli/sfr'
   } else {
     return `https://${session.storeFqdn}`
   }
 }
 
-function isThemeAccessSession(session: DevServerSession) {
+function isThemeAccessSession(session: AdminSession) {
   return session.token.startsWith('shptka_')
 }
```

Comment:
This is pretty trivial, but what do you think about moving this to `dev-server-session`?